index.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     'valid-jsdoc': ['error'],
     'backpack/use-tokens': 'error',
     'backpack/auto-import-tokens': 'error',
+    'react/destructuring-assignment': 'off',
```

Comment:
Is it worth adding a comment here? We generally leave one when turning a rule off, just so people know why.